assets/src/edit-story/elements/text/util.js

```diff
@@ -43,7 +45,11 @@ export function generateParagraphTextStyle(
   };
 }
 
-export const generateFontFamily = ({ family, fallbacks }) => {
+export const generateFontFamily = (font) => {
+  if (!font) {
+    return 'inherit';
+  }
+  const { family, fallbacks } = font;
```

Comment:
This could also be fixed with a default argument:

```js
export const generateFontFamily = ({ family, fallbacks } = {}) => {
```

Reply:
Then it will produce `null` (so `font-family: null`) - I was wondering what we should return if the font is undefined, my version returns `inherit` (not sure if useful) in that case.
Either way, I will look more into this function because both versions can produce something like `font-family: nullmonospace`.

Reply:
Done in https://github.com/google/web-stories-wp/pull/3184/commits/32ef66a82be3168fad1ed34d03a2ebb515fcb6b9